src/cmake-tools.ts

```diff
@@ -761,7 +761,7 @@ export class CMakeTools implements vscode.Disposable, api.CMakeToolsAPI {
       }
 
       const sourceDirectory = (await this.sourceDir).toLowerCase();
-      if (str === path.join(sourceDirectory, "cmakelists.txt")) {
+      if ((str.includes("cmakelists.txt") || str.includes(".cmake"))) {
```

Comment:
instead of `includes`, I think `endsWith` would be better.

Reply:
I made a small change, we want only the cmakelists that are in the workspace folder or in the sourcedirectory.